app/src/main/java/fr/neamar/kiss/loader/LoadContactsPojos.java

```diff
@@ -60,10 +88,9 @@ public LoadContactsPojos(Context context) {
                     if (contact.phone == null) {
                         contact.phone = "";
                     }
-                    contact.phoneSimplified = contact.phone.replaceAll("[-.(): ]", "");
-
-                    contact.homeNumber = homePattern.matcher(contact.phone).lookingAt();
-
+                    contact.phoneSimplified = contact.phone.replaceAll("[-.():/ ]", "");
```

Comment:
I know it was like that before, but maybe we can `Pattern.compile` once to avoid compiling the regexp on each contact?